www/core/test/list_test.xml

```diff
@@ -1,7 +1,7 @@
 <extensionset name="Joomla Core Test Updateserver" description="The Joomla Core Updateserver for Tests of Alpha, Beta und RC Releases">
```

Comment:
Thanks original this was all german so i have missed this on taking over here. Thanks for spotting